code/game/gamemodes/umbrage/umbrage_datum.dm

```diff
@@ -9,13 +9,17 @@
 	var/cycle_progress = 0 //When this reaches 5, it will reset to 0 and regenerate up to 20 spent psi.
 	var/lucidity = 3 //Lucidiy is used to buy abilities. We gain one each time we drain someone.
 	var/lucidity_drained = 3 //How much lucidity we've taken overall.
+	var/list/ability_types = list() //Types of abilities that the umbrage CAN OWN
```

Comment:
move this to the ```New()```, it avoids byond's hidden ```init()``` procs that are more costly than we like.